Happy Dance/Assets/Fader/Fader.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEngine.Assertions;
+
+/// <summary>
+/// Fader is used to controller the color of a target material. It is driven by an animation clip.
+/// </summary>
+public class Fader 
```

Comment:
The fader material relies on "_Tint" property but not all materials has tint. Might be good to A) expose "_Tint" so the property target can be configured or B) some kind of Assert to ensure the assigned material has a "_Tint".

Reply:
Done